topics:
  inputs:
    arm_state:
      name: /arm/joint_state
      type: sensor_msgs/JointState
    chassis_state:
      name: /chassis/chassis_state
      type: sensor_msgs/JointState
    target_list:
      name: /router/target_list
      type: geometry_msgs/PoseStamped
  outputs:
    status:
      name: /task_mgr/status
      type: task_mgr/task_mgr